assets/src/application/ui/input/Button/ButtonGroup/style.scss

```diff
@@ -3,17 +3,21 @@
 .ee-btn-group.ee-btn-group {
 	border-radius: var(--ee-border-radius-small);
 	box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.35), -1px -1px 2px rgba(255, 255, 255, 0.95);
-	margin: var(--ee-margin-nano);
+	margin: var(--ee-margin-nano) var(--ee-margin-smaller) var(--ee-margin-smaller) var(--ee-margin-nano);
+
+	[dir='rtl'] & {
+		margin: var(--ee-margin-nano) var(--ee-margin-nano) var(--ee-margin-smaller) var(--ee-margin-smaller);
```

Comment:
cool :+1: 
nice and clean approach

Reply:
![image](https://user-images.githubusercontent.com/1477453/84484547-93f25380-aca3-11ea-8e55-25ca89d31228.png)